src/components/report/ReportChapterTableResult.svelte

```diff
@@ -59,7 +59,7 @@
       <ul>
         {#each criteria.components as component}
         {#if component.adherence.notes}
-          <li>{@html catalogComponentLabel(catalogName, component.name, "html")}{@html sanitizeMarkdown(component.adherence.notes)}</li>
+          <li><!-- {@html catalogComponentLabel(catalogName, component.name, "html")} -->{@html sanitizeMarkdown(component.adherence.notes)}</li>
```

Comment:
I couldn't help but notice that removing the label displays the conformance level with no context. Screenshot: 
![Screenshot of a criteria example with conformance level missing web, electronic documents, software, authoring tool](https://user-images.githubusercontent.com/7096681/215611523-6542fdd0-6ce1-4349-980e-784e0015c226.png)

Is there another display we should try to keep those labels? I wanted to avoid the table in a table look so open to ideas.

Reply:
@maggiewachs ☝🏾

Reply:
@dmundra I think the only issue here is that the labels need to be conditional. If we're only assessing web criteria, for example, we don't want to see "Web:" precede every conformance level and comment. But if you're assessing web and software, the labels are needed to differentiate. Does that make sense?

Reply:
Ah great point. I will add that conditional check.

Reply:
In https://github.com/CivicActions/openacr-editor/pull/13 I have the solution to hide labels if only one level is selected.